packages/react-charting/package.json

```diff
@@ -17,15 +17,16 @@
     "build": "just-scripts build",
     "bundle": "just-scripts bundle",
     "lint": "just-scripts lint",
-    "test": "just-scripts test",
+    "test": "cross-env TZ=UTC just-scripts test",
     "just": "just-scripts",
     "clean": "just-scripts clean",
     "codepen": "node ../../scripts/executors/local-codepen.js",
     "code-style": "just-scripts code-style",
     "start": "just-scripts dev:storybook",
     "start:legacy": "just-scripts dev",
     "start-test": "just-scripts jest-watch",
-    "update-snapshots": "just-scripts jest -u"
+    "update-snapshots": "cross-env TZ=UTC just-scripts jest -u",
+    "test:tz": "node ./run-tests-in-timezone.js"
```

Comment:
>node [](http://example.com/codeflow?start=16&length=4)

we are using yarn instead of node now

Reply:
Tried it but it's not working. I think yarn is like npm and can be used to run scripts, but node is required to run js files. The above `codepen` script is like this only.